nomad/state/events_test.go

```diff
@@ -1055,6 +1056,58 @@ func Test_eventsFromChanges_ACLRole(t *testing.T) {
 	require.Equal(t, aclRole, eventPayload.ACLRole)
 }
 
+func Test_eventsFromChanges_ACLAuthMethod(t *testing.T) {
+	ci.Parallel(t)
+	testState := TestStateStoreCfg(t, TestStateStorePublisher(t))
+	defer testState.StopEventBroker()
+
+	// Generate a test ACL auth method
+	authMethod := mock.ACLAuthMethod()
+
+	// Upsert the auth method straight into state
+	writeTxn := testState.db.WriteTxn(10)
+	updated, err := testState.upsertACLAuthMethodTxn(10, writeTxn, authMethod)
+	must.True(t, updated)
+	must.NoError(t, err)
+	writeTxn.Txn.Commit()
+
+	// Pull the events from the stream.
+	upsertChange := Changes{Changes: writeTxn.Changes(), Index: 10, MsgType: structs.ACLAuthMethodsUpsertRequestType}
+	receivedChange := eventsFromChanges(writeTxn, upsertChange)
+	must.NotNil(t, receivedChange)
+
+	// Check the event, and it's payload are what we are expecting.
```

Comment:
```suggestion
	// Check the event, and its payload are what we are expecting.
```
There's another one below as well

Reply:
🤦 English is so hard.

Reply:
:nod-sob: